popup.el

```diff
@@ -1363,7 +1378,11 @@ isearch canceled. The arguments is whole filtered list of items."
                            :parent-offset parent-offset))
   (unwind-protect
       (progn
-        (popup-set-list menu list)
+        (popup-set-list menu
+                        (cl-loop with idx = 0
+                                 for e in list
+                                 collect (popup-make-item e :rawindex idx)
+                                 do (incf idx)))
```

Comment:
This can also be even simpler:
```emacs-lisp
(cl-loop for e in list
         for idx from 0
         collect (popup-make-item e :rawindex idx))
```